### PR TITLE
[3gpk47v] blog homepage redesign

### PR DESCRIPTION
**Pull request by pebiantara**
[Blog Homepage redesign](https://app.clickup.com/t/3gpk47v)

### CODE REVIEW

File: src/scss/core/_utilities.scss [hunk 1]
```diff
@@ -11,11 +11,22 @@
 @include bg-variant(".bg-gray-100", $gray-100, true);
 @include bg-variant(".bg-gray-400", $gray-400, true);
 
+// Position
+.top-30 {
```
Comment (foria) [inline src/scss/core/_utilities.scss:15]: these are customizations of the original carousel, if original carousel has changed we need to talk with the designer, asking why

File: src/scss/core/_utilities.scss [hunk 2]
```diff
@@ -11,11 +11,22 @@
...
+}
+
 // Border color
 .border-bg-primary-light-second {
 	border-color: $primary-light-second !important;
 }
 
+
+// Border gray
+.border-bg-gray-300 {
```
Comment (foria) [inline src/scss/core/_utilities.scss:26]: same as above

**Review by foria — CHANGES_REQUESTED** [review]
check comments

File: src/scss/core/_utilities.scss (hunk 1 above)
Comment (pebiantara) [reply src/scss/core/_utilities.scss:15]: let me double check, looks like we have carousel centered on image prev/next button for products carousel in the homepage

File: src/scss/core/_utilities.scss (hunk 2 above)
Comment (pebiantara) [reply src/scss/core/_utilities.scss]: updated with secondary light color

File: src/scss/core/_utilities.scss (hunk 1 above)
Comment (pebiantara) [reply src/scss/core/_utilities.scss]: updated code following same way with product carousel

**Review by foria — APPROVED** [review]

**Pull request merged**